### PR TITLE
[ 🚧 WIP 🚧 ] Github Actions Testing

### PR DESCRIPTION
**Pull request by lynncyrin**
[ 🚧 WIP 🚧 ]

---

Relates to https://github.com/urfave/cli/issues/923

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
```
Comment (asahasrabuddhe) [inline .github/workflows/go.yml]: Why do we need this?

Comment (lynncyrin) [reply .github/workflows/go.yml]: This whole file was copied from the Github Actions golang setup, and I haven't edited it yet.

Which is to say: 🤷‍♀

Comment (saschagrunert) [reply .github/workflows/go.yml]: I think we can remove this :)

Comment (lynncyrin) [reply .github/workflows/go.yml]: To be super clear: this is a draft PR, I literally just copy pasted all of this. Pleeeeeease wait until it's ready before making comments.

**Comment by lynncyrin** [comment]
I'm gonna close this - so that someone else can work on it if they want!

**Pull request closed without merging**